src/test/java/com/skhu/luxuryshop/user/encoder/BCryptPasswordEncoderTest.java

```diff
@@ -0,0 +1,33 @@
+package com.skhu.luxuryshop.user.encoder;
+
+import com.skhu.luxuryshop.user.dto.UserResponseDto;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BCryptPasswordEncoderTest {
+    BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+    String originPassword = "password";
+    String encryptedPassword = "$2a$10$MafzWKqMEq8R1iYIydZ7nOu02R7DwSKdf5bEuqbimtu/mt8eM3N/q";
+
+    @DisplayName("findById_존재하는 유저의 id인 경우")
+    @Test
+    void test_encrypt() {
+        String encryptedPassword = passwordEncoder.encrypt(originPassword);
+
+        assertThat(true).isEqualTo(passwordEncoder.isMatch(originPassword, encryptedPassword));
```

Comment:
```suggestion
        assertThat(passwordEncoder.isMatch(originPassword, encryptedPassword)).isTrue()
```
assertThat에 인자로 검증하고자하는 걸 넣고 뒤에  isEqualTo나 isTrue를 하는게 더 좋겠어요.
AssertJ에는 isTrue 같은 상황에 맞는 메서드가 많으니 API 문서를 한 번 보는것도 좋을 것 같아요!